lib/JITIDL/ChakraJIT.idl

```diff
@@ -3,8 +3,13 @@
 // Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
 //-------------------------------------------------------------------------------------------------------
 
+cpp_quote("#ifndef __JITTypes_h__")
+cpp_quote("#define __JITTypes_h__")
```

Comment:
midl does not happy with that

---
In reply to: [92259422](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92259422) [](ancestors = 92259422,92259202,92255395)